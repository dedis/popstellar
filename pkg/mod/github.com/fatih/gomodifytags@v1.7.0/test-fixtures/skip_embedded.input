package event

import "time"

type StationEvent interface {
	DomainEvent
	isStationEvent()
}

type stationEvent struct{}

func (evt stationEvent) isStationEvent() {}

type StationCreated struct {
	stationEvent
	ConnectorID    int
	MeterStop      int
	TransactionID  int32
	ChargingCardID identifier.ChargingCardID
	Timestamp      time.Time `json:"timestamp`
}
