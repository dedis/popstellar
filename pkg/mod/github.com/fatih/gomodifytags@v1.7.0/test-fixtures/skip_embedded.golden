package event

import "time"

type StationEvent interface {
	DomainEvent
	isStationEvent()
}

type stationEvent struct{}

func (evt stationEvent) isStationEvent() {}

type StationCreated struct {
	stationEvent
	ConnectorID    int                       `json:"connector_id"`
	MeterStop      int                       `json:"meter_stop"`
	TransactionID  int32                     `json:"transaction_id"`
	ChargingCardID identifier.ChargingCardID `json:"charging_card_id"`
	Timestamp      time.Time                 `json:"timestamp`
}
